Employee.h

```diff
@@ -0,0 +1,14 @@
+#import "Organization.h"
+#import "Person.h"
+@interface Employee: Person{
+    Organization *organization;
+    NSNumber *employeeId ;
+}
+//@property(readonly) NSString *fullName;
```

Comment:
Keep the property for fullname and override its getter instead of creating a separate method for getting the property.